catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/operation/impl/QueryImpl.java

```diff
@@ -163,6 +162,21 @@ public void setRequestsTotalResultsCount(boolean requestsTotalResultsCount) {
 
   @Override
   public String toString() {
-    return ToStringBuilder.reflectionToString(this);
+    StringBuilder sb = new StringBuilder();
```

Comment:
✏️ I think `String.format` would look more readable.

Reply:
For classes with multiple fields, I personally prefer this way for readability over the String format. 🤷‍♂️

Reply:
Same as above, as I wanted to make sure these methods were as performant as possible.